Assets/MRTK/Core/Providers/BaseController.cs

```diff
@@ -234,50 +234,50 @@ public void AssignControllerMappings(MixedRealityInteractionMapping[] mappings)
         /// <returns>True if a model was successfully loaded or model rendering is disabled. False if a model tried to load but failed.</returns>
         protected virtual bool TryRenderControllerModel(Type controllerType, InputSourceType inputSourceType)
         {
-            if (GetControllerVisualizationProfile() == null ||
-                !GetControllerVisualizationProfile().RenderMotionControllers)
+            MixedRealityControllerVisualizationProfile controllerVisualizationProfile = GetControllerVisualizationProfile();
+            bool controllerVisualizationProfilePresent = GetControllerVisualizationProfile() != null;
```

Comment:
Can this reuse the cached `controllerVisualizationProfile`?

```suggestion
            bool controllerVisualizationProfilePresent = controllerVisualizationProfile != null;
```